### PR TITLE
EV-129: Created login component

### PR DESCRIPTION
**Pull request by xcaho**

### CODE REVIEW

File: src/app/participant/survey-registration/survey-registration.component.ts [hunk 1]
```diff
@@ -66,21 +66,21 @@ export class SurveyRegistrationComponent {
...
       this.survey = surveyDto
-      this.fetchEvent(this.survey.eventId);
+      this.fetchEvent(this.survey.eventId, true);
     }, (error) => {
       this.router.navigate(['/404'])
     })
   }
 
-  private fetchEvent(eventId: number): void {
+  private fetchEvent(eventId: number, navigate: boolean): void {
```
Comment (michalnowak18) [inline src/app/participant/survey-registration/survey-registration.component.ts:75]: Rename this to shouldNavigate

File: src/app/participant/survey-registration/survey-registration.component.ts [hunk 2]
```diff
@@ -121,6 +121,11 @@ export class SurveyRegistrationComponent {
 
         if (exception.status === 409) {
           this.alertService.showError('Wybrany termin już jest zajęty, wybierz inny lub odśwież stronę.');
+          this.fetchEvent(this.survey.eventId, false);
```
Comment (michalnowak18) [inline src/app/participant/survey-registration/survey-registration.component.ts:124]: Extract inside of this if to a separate private function, i.e handleHttp409()

File: src/app/participant/survey-registration/survey-registration.component.ts (hunk 1 above)
Comment (xcaho) [reply src/app/participant/survey-registration/survey-registration.component.ts]: done

File: src/app/participant/survey-registration/survey-registration.component.ts [hunk 3]
```diff
@@ -66,13 +66,13 @@ export class SurveyRegistrationComponent {
...
       this.survey = surveyDto
-      this.fetchEvent(this.survey.eventId, true);
+      this.shouldNavigate(this.survey.eventId, true);
     }, (error) => {
       this.router.navigate(['/404'])
     })
   }
 
-  private fetchEvent(eventId: number, navigate: boolean): void {
+  private shouldNavigate(eventId: number, navigate: boolean): void {
```
Comment (michalnowak18) [inline src/app/participant/survey-registration/survey-registration.component.ts:75]: A small misunderstanding here, I meant that the parameter "navigate" should be named "shouldNavigate".
Sorry for confusion! 😀

Comment (xcaho) [reply src/app/participant/survey-registration/survey-registration.component.ts:75]: Oh my god

**Review by michalnowak18 — APPROVED** [review]

**Pull request merged**